support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -23,6 +25,13 @@ type Props = {
 	abParticipations: Participations;
 };
 
+const ReturnAddressSchema = object({
+	link: string(),
+});
+export function setReturnAddress(link: InferInput<typeof ReturnAddressSchema>) {
+	storage.session.set('returnAddress', link);
```

Comment:
Do we need permission from Data privacy to add a new item to session storage?

Reply:
Good spot, have applied on the form, thanks for that.